packages/components/pages/f-contact-preferences/src/components/ContactPreferences.vue

```diff
@@ -313,21 +315,18 @@ export default {
 };
 </script>
 
-<style lang="scss">
-.c-formField-label,
-.c-formField-label-description {
-    margin-bottom: 0;
-}
-</style>
-
 <style lang="scss" module>
-
 .c-contactPreferences-fieldset {
     display: flex;
     flex-flow: column;
     border: none;
     padding: 0;
-    margin: spacing(x2) 0;
+    margin: spacing(x2) 0 spacing(x4);
+}
+
+.c-formField-label,
```

Comment:
Won't moving these into the `module` block mean that it'll get scoped and not apply to the `formField` component?

Reply:
Good spot. It's actually no longer needed, though. Removing it :)